src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObj = {};
+
+  const stylesArr = sourceString.split(';')
+    .map(style => style.split(':'));
+
+  stylesArr.forEach(style => {
+    if (style.length === 2) {
+      const kay = style[0].trim();
+      const value = style[1].trim();
+
+      stylesObj[kay] = value;
+    }
+  });
```

Comment:
Actually, it would be better to use `reduce` here, instead of forEach. Let's rewrite this solution using reduce